addons/l10n_ch/data/account_vat2011_data.xml

```diff
@@ -11,6 +11,7 @@
             <field name="chart_template_id" ref="l10nch_chart_template"/>
             <field name="type_tax_use">sale</field>
             <field name="tax_group_id" ref="tax_group_tva_25"/>
+            <field name="active" eval="False"/>
```

Comment:
@clbr-odoo I am afraid that if we do this already, new dbs and also old ones that upgrade maybe, will have the old taxes activated while they should still be used in 2023?